src/errors/test-run/templates.js

```diff
@@ -78,12 +78,31 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.selectActionSelectorTypeError]: err => markup(err, `
+        The ${err.argumentName} argument is expected to be a string, but it was <code>${err.actualType}</code>.
```

Comment:
Shouldn't it be `The ${err.argumentName} selector is expected...`? We can combine it with `dragDestinationSelectorTypeError` and name something like `actionaAddtionalSelectorTypeError` where we pass selector name, like `destination`, etc. (it should same name as in API description and wrapped in `<code>`)